Resources/Computing/PoolComputingElement.py

```diff
@@ -87,12 +107,22 @@ def submitJob( self, executableFile, proxy, **kwargs ):
       pilotProxy = ret['Value']['path']
     self.log.notice( 'Pilot Proxy:', pilotProxy )
 
+    kwargs = { 'UseSudo': False }
+    if self.useSudo:
+      for nUser in range( 16 ):
```

Comment:
Well, it is just a "large" number of users on the node. VM defines a number of unix users that are running payloads with sudo. The total number of users normally should be no more than the number of processors.